docs/sources/visualizations/candlestick.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Candlestick"
+description = "Candlestick visualization documentation"
+keywords = ["grafana", "Candlestick", "OHLC", "panel", "documentation"]
+aliases = ["/docs/grafana/latest/features/panels/candlestick/", "/docs/grafana/latest/panels/visualizations/candlestick/"]
+weight = 600
++++
+
+# Candlestick
+
+The Candlestick panel allows you to visualize data which includes a number of consistent dimensions focused around price movement. The Candlestick panel also includes an Open-High-Low-Close (OHLC) mode, as well as support for additional dimensions based on time series data.
+
+{{< figure src="/static/img/docs/candlestick-panel/candlestick-panel-8-3.png" max-width="1200px" caption="Candlestick panel" >}}
+
+The Candlestick panel builds upon the foundation of the [time series]({{< relref "./time-series/_index.md" >}}) panel and includes many common configuration settings.
+
+## Mode
+
+The mode options allow you to toggle which dimensions are used for the visualization.
+
+- **Candles** limits the panel dimensions to the open, high, low and close dimensions used by candlestick visualizations.
+- **Volume** limits the panel dimension to the volume dimension.
+- **Both** is the default behavior for the candlestick panel. It includes both candlestick and volume visualizations.
+
+## Candle style
+
+- **Candles** is the default display style and creates candle-style visualizations between the open and close dimensions.
+- **OHLC Bars** displays the four core dimensions open, high, low and close values.
+
+## Color strategy
+
+- **Since Open** is the default behavior. This mode will utilize the _Up_ color (below) if the intra-period price movement is positive. In other words, if the value on close is greater than the value on open, the _Up_ color is used.
+- **Since Prior Close** is an alternative display method which will utilize the _Up_ color (below) if the inter-period price movement is positive. In other words, if the value on open is greater than the previous value on close, the _Up_ color is used.
```

Comment:
```suggestion
- **Since Prior Close** is an alternative display method which will utilize the _Up_ color (below) if the inter-period price movement is positive. In other words, if the value on close is greater or equal than the value on previous close, the _Up_ color is used.
```

Reply:
we should also note that **Since Prior Close** also toggles the hollow candles style. where their hollow = an intra (since open) Up or same, and filled represents an intra (since open) Down.

this explains the bizarre differences, and might be worth linking from our docs: https://thetradingbible.com/how-to-read-hollow-candlesticks

Reply:
Thanks - I'll update and re-ping for a quick check.